integration-tests/experimental/callbacks/test_langchain_tracer.py

```diff
@@ -0,0 +1,140 @@
+import json
+from typing import Type
+
+import pytest
+from langchain import LLMChain, PromptTemplate
+from langchain.agents import AgentType, initialize_agent
+from langchain.agents.agent import AgentExecutor
+from langchain.chains.openai_functions import create_openai_fn_chain
+from langchain.chat_models import ChatOpenAI
+from langchain.llms import BaseLLM, OpenAI
+from langchain.tools import tool
+from phoenix.experimental.callbacks.langchain_tracer import OpenInferenceTracer
+from phoenix.trace.schemas import SpanKind
+from pydantic import BaseModel, Field
+
+
+@pytest.fixture
+def tracer() -> OpenInferenceTracer:
+    """An OpenInference tracer"""
+    return OpenInferenceTracer()
+
+
+@pytest.mark.parametrize(
+    "llm_class,model_name",
+    [
+        pytest.param(OpenAI, "gpt-3.5-turbo", id="openai-llm"),
+        pytest.param(ChatOpenAI, "gpt-3.5-turbo", id="openai-chat-model"),
+    ],
+)
+def test_tracer_records_common_llm_attributes_for_llm_chain(
```

Comment:
I think this one should shift to a unit test as it doesn't need the LLM call - stubbing the API call should do. Not blocking though.

Reply:
This one is tricky, requires multiple API stubs (one for the embeddings call, another for the LLM call). Going to punt for now.